[docs] Update Deckhouse CLI installation guide (#12855)
deckhouse/deckhouse: docs/documentation/pages/DECKHOUSE-CLI.md

    </details>
    </div>

## How do I install the Deckhouse CLI?

There are two ways to install the Deckhouse CLI:

* Starting with version 0.10, you can install using [trdl](https://trdl.dev/). This method allows you to get fresh versions of the tool with all improvements and fixes.
  > Note that trdl installation requires Internet access to the tuf repository containing the tool. This method will not work in an air-gapped environment!
* You can manually download the executable file and install it on the system.

### trdl-based installation

Starting with the Deckhouse CLI version 0.10, you can install it using [trdl](https://trdl.dev/).

{% alert %}

> When installing within the cluster, enable [Deckhouse Tools](../modules/deckhouse-tools/) and follow the instructions provided in the interface.

Author: Deckhouse tools has nothing to do with trdl

1. Add the Deckhouse CLI repository to trdl:

   ```bash
   URL=https://deckhouse.ru/downloads/deckhouse-cli-trdl
   ROOT_VERSION=1
   ROOT_SHA512=343bd5f0d8811254e5f0b6fe292372a7b7eda08d276ff255229200f84e58a8151ab2729df3515cb11372dc3899c70df172a4e54c8a596a73d67ae790466a0491
   REPO=d8

   trdl add $REPO $URL $ROOT_VERSION $ROOT_SHA512
   ```

1. Install latest stable release and validate that the `d8` binary is functional:

   ```bash
   . $(trdl use d8 0 stable) && d8 --version
   ```

> If you dont want to call `. $(trdl use d8 0 stable)` every time you need to use Deckhouse CLI, consider adding `alias d8='trdl exec d8 0 stable -- "$@"'` to your shell RC file.

Reviewer: ```suggestion

If you don't want to call `. $(trdl use d8 0 stable)` every time you need to use Deckhouse CLI, consider adding `alias d8='trdl exec d8 0 stable -- "$@"'` to your shell RC file.
```

1. Add the Deckhouse CLI repository to trdl:

   ```bash
   URL=https://deckhouse.ru/downloads/deckhouse-cli-trdl
   ROOT_VERSION=1
   ROOT_SHA512=343bd5f0d8811254e5f0b6fe292372a7b7eda08d276ff255229200f84e58a8151ab2729df3515cb11372dc3899c70df172a4e54c8a596a73d67ae790466a0491
   REPO=d8

   trdl add $REPO $URL $ROOT_VERSION $ROOT_SHA512
   ```

> 1. Install latest stable release and validate that the `d8` binary is functional:

Reviewer 2: ```suggestion
1. Install the latest stable release of the `d8` utility and verify its functionality:
```

   ```bash
   . $(trdl use d8 0 stable) && d8 --version
   ```

> If you don't want to call `. $(trdl use d8 0 stable)` every time you need to use Deckhouse CLI, consider adding `alias d8='trdl exec d8 0 stable -- "$@"'` to your shell RC file.

Reviewer 2: ```suggestion
If you don't want to call `. $(trdl use d8 0 stable)` every time you need to use Deckhouse CLI, add the following line to your shell’s RC file: `alias d8='trdl exec d8 0 stable -- "$@"'`.
```
